.github/workflows/ci.yml

```diff
@@ -138,11 +138,13 @@ jobs:
       - name: Select Toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
what's the reason for this change?

Reply:
we build everything with nightly... this, I think, was an oversight

Reply:
ah I see